src/Symfony/Component/Security/Core/Tests/Authentication/Provider/DaoAuthenticationProviderTest.php

```diff
@@ -83,7 +83,7 @@ public function testRetrieveUserReturnsUserFromTokenOnReauthentication()
         $provider = new DaoAuthenticationProvider($userProvider, $this->getMockBuilder('Symfony\\Component\\Security\\Core\\User\\UserCheckerInterface')->getMock(), 'key', $this->getMockBuilder('Symfony\\Component\\Security\\Core\\Encoder\\EncoderFactoryInterface')->getMock());
         $reflection = new \ReflectionMethod($provider, 'retrieveUser');
         $reflection->setAccessible(true);
-        $result = $reflection->invoke($provider, null, $token);
```

Comment:
💡 `null` shouldn't be a valid argument for the `retrieveUser` method. The test only worked because the parameter is ignored on the execution path that is tested here.